runtime/centrifuge/src/weights/mod.rs

```diff
@@ -23,6 +23,7 @@ pub mod pallet_migration_manager;
 pub mod pallet_multisig;
 pub mod pallet_preimage;
 pub mod pallet_proxy;
+pub mod pallet_restricted_tokens;
```

Comment:
We have been using the weights from the pallet not from the runtime for `centrifuge`. Not sure if this was the intended behavior. I've changed it in order to use the correct weights from now